applications/cheap_flights/data_manager.py

```diff
@@ -34,26 +34,27 @@ def update_airline_name(airline_id) -> str:
         return airline_id
 
 
-def format_flight_route(flight_routes) -> str:
+def format_flight_route(flight_routes) -> list[str]:
     """ Format the flight route information into a string """
 
     def format_datetime(datetime_str):
-        date = datetime.fromtimestamp(datetime_str)
         """ Format datetime object as string 'dd/mm/yy at 00:00 hours'"""
+        date = datetime.fromtimestamp(datetime_str)
         return date.strftime('%d/%m/%y at %H:%M')
 
-    formatted_text = []
+    formatted_routes = []
     for route in flight_routes:
-        fly_from = route["flyFrom"]
-        city_from = route["cityFrom"]
-        fly_to = route["flyTo"]
-        city_to = route["cityTo"]
+        airline = update_airline_name(route["airline"])
         local_departure = format_datetime(route["dTime"])
         local_arrival = format_datetime(route["aTime"])
-        flight_no = route["flight_no"]
-        airline = update_airline_name(route["airline"])
 
-        formatted_text.append(
-            f"{airline} | Flight number: {flight_no} - {fly_from}({city_from}) to {fly_to}({city_to}) - Departure: {local_departure} - Arrival: {local_arrival}. ")
-    return formatted_text
+        formatted_route = (f"{airline} | "
+                           f"Flight number: {route['flight_no']} - "
+                           f"{route['flyFrom']}({route['cityFrom']}) to {route['flyTo']}({route['cityTo']}) - "
+                           f"Departure: {local_departure} - "
+                           f"Arrival: {local_arrival}. ")
```

Comment:
This is one way to avoid a very long string.  It's kind of a cheat because python will combine back to back strings like this.  It might be easier to read like this instead of one very long line.  Or maybe not.  Up to you.

Reply:
agreed 100%

edited.